### ISSUE TITLE
Hi, are BN layers in classifier and mask network helipful?

### ISSUE THREAD

**Issue by mks0601** [tensorboy/Pytorch_Mask_RCNN#1]
Hi, thanks for sharing your code.

You converted fpn_classifier_graph to fully convolutional manner and added BN layers.
Also, in build_fpn_mask_graph, you also added BN layers which is not mentioned in papers (they mentions only ReLU layers).

Does adding BN layers beneficial for the performance of the model?

**Comment by tensorboy** [comment]
Hi, @mks0601. As we don't have a complete training pipeline yet, the model was converted. To let the converted model as less as possible to sacrifice the performance, I also included the BN layers. But I don't think BN layers will help because the batch size of the training is small usually for mask rcnn. :)
